domains/rioirsan1.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "yohanesrioirsan",
+        "email": "rioirsan8@gmail.com"
+    },
+    "record": {
+        "CNAME": "yohanesrioirsan.vercel.app"
```

Comment:
Invalid. Vercel verification records should be under files _vercel.domain-request.json, not the root file. See the docs for more information on how to setup a Vercel Site. https://docs.is-a.dev/guides/vercel